kolibri/plugins/learn/assets/src/views/HybridLearningContentCardListView.vue

```diff
@@ -1,86 +1,59 @@
 <template>
 
-  <div
-    class="card drop-shadow"
-    :class="[
-      { 'mobile-card': isMobile }
-    ]"
-  >
-    <router-link
-      :to="link"
+  <router-link :to="link" :class="$computedClass({ ':focus': $coreOutline })">
+    <div
       class="card"
-      :class="[
-        { 'mobile-card': isMobile },
-        $computedClass({ ':focus': $coreOutline })
-      ]"
+      :class="{ 'mobile-card': isMobile }"
       :style="{ backgroundColor: $themeTokens.surface }"
     >
-      <div
-        v-if="!isMobile"
-        class="folder-header"
-        :style="{ backgroundColor: (!content.is_leaf ? $themeTokens.text : null ) }"
-      ></div>
-      <div class="thumbnail">
-        <CardThumbnail
-          :isMobile="isMobile"
-          :contentNode="content"
-        />
-      </div>
-      <span class="details" :style="{ color: $themeTokens.text }">
-        <div
-          class="metadata-info"
-          :style="{ color: $themePalette.grey.v_700 }"
-        >
-          <LearningActivityLabel
-            :contentNode="content"
-            :labelAfter="true"
-            :hideDuration="!isMobile"
-            condensed
-          />
-        </div>
-        <h3 class="title">
-          <TextTruncator
-            :text="content.title"
-            :maxHeight="maxTitleHeight"
+      <KFixedGrid numCols="4">
+        <KFixedGridItem :span="isMobile ? 4 : 1" class="thumb-area">
+          <CardThumbnail
+            :kind="content.kind"
+            v-bind="{ thumbnail, isMobile }"
+            :activityLength="content.duration"
           />
-        </h3>
-        <p class="text">
-          <TextTruncator
-            :text="content.description"
-            :maxHeight="maxDescriptionHeight"
-          />
-        </p>
-        <div v-if="displayCategoryAndLevelMetadata && !isMobile" class="metadata-info">
-          <p> {{ coreString(displayCategoryAndLevelMetadata) }}</p>
-        </div>
```

Comment:
this is probably a regression. When I deleted at first, it looked like a bug to me. Now I'm realizing it might be handling cases that I don't understand.

Can revisit tomorrow

Reply:
fixed